opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/aggregation.py

```diff
@@ -176,27 +176,25 @@ def aggregate(self, measurement: Measurement) -> None:
                 self._value = 0
             self._value = self._value + measurement.value
 
-    def collect(self) -> Optional[Sum]:
+    def collect(self, collection_start: int) -> Optional[Sum]:
```

Comment:
We need to align the initial start times still somehow (on L164). Maybe at instantiation we can pass that in?

Reply:
Fixed